recipe/meta.yaml

```diff
@@ -28,12 +28,12 @@ requirements:
     - pip
     - setuptools
     - wheel
-    - libsodium =1.0.18
-    - zeromq =4.3.4
+    - libsodium >=1.0.18,<1.0.19.0a0
+    - zeromq >=4.3.4,<4.4.0a0
```

Comment:
`conda-build` always picks up the latest version of the package if it will find the range of versions in `host`.
So the correct pinning in `host` is:
```
    - libsodium 1.0.18
    - zeromq 4.3.4
```
In this case in `run` must be:
```
    - libsodium >=1.0.18,<1.0.19.0a0
    - zeromq >=4.3.4,<4.4.0a0
```

Reply:
Thank you for pointing that out, I misunderstood what you were saying in the first comment.